File: israaAdmin/app/Models/Circuit.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Circuit extends Model
{
    use HasFactory;
    protected $fillable = [
        'image',
        'name',
        'nbjours',
        'description',
        'date',
        'prix',
        
    ];
}

File: day34/lib/models/circuit.dart
class Circuit {
  final int id;
  final String image;
  final String name;
  final String prix;
  final String description;
  final String nbjours;
  final DateTime date;
  Circuit(this.id, this.image, this.name, this.prix, this.description,
      this.nbjours, this.date);

  factory Circuit.fromJson(Map<String, dynamic> json) {
    return new Circuit(
        json["id"],
        json["image"],
        json["name"],
        json["prix"].toString(),
        json["description"],
        json["nbjours"],
        DateTime.parse(json["date"]));
  }
}
